df.world-site.xml

```diff
@@ -431,7 +431,14 @@
             <static-array count='51' pointer-type='site_realization_crossroads'/>
         </static-array>
         <static-array name='river_map' count='51'>
-            <static-array count='51' type-name='pointer'/>
+            <pointer>
+                <static-array name='start' type-name='int32_t' count='48'/>
+                <static-array name='end'   type-name='int32_t' count='48'/>
+                <int32_t name="unk_180"/>
+                <int32_t name="unk_184"/>
+                <int32_t name="start_array_limits"/>
```

Comment:
What's this? It's kind of unclear from the name, so maybe an explanation in a comment would be helpful.